src/integration-tests/bash/job.sh

```diff
@@ -1,15 +1,15 @@
 #!/bin/bash
-# Copyright 2017, 2018, Oracle Corporation and/or its affiliates. All rights reserved.
+# Copyright 2017, 2019, Oracle Corporation and/or its affiliates. All rights reserved.
 # Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
 
 #
 # job.sh <command> [<optarg1>] [<optarg2>] ...
 #
-#    This is a helper script called by run.sh & cleanup.sh.  It runs the given bash
```

Comment:
This helper script may need to be removed as well if we confirm that we need to remove cleanup.sh.

Reply:
The job.sh is also called in java test.

Reply:
okay.